### ISSUE TITLE
Non-ascii (unicode) characters are lost in PDF and Word output

### ISSUE THREAD

**Issue by mdingemanse** [crsh/papaja#209]
If I knit an Rmd document that includes UTF-8 characters outside the ASCII range (for instance, phonetic symbols like ɛ or ɔ), they never survive the transition to PDF (with `output: papaja::apa6_pdf`) or to .docx (with `output: papaja::apa6_word`).

Inspecting the intermediate files generated on the way to a PDF, it looks like the characters are still there in full UTF8 glory in the .md file, and they're converted to codepoints like `<U+0254>` in .utf8.md. In the PDF, they are either invisible (when embedded in a paragraph) or show up as the unparsed codepoints (e.g. `<U+0254>`) in the PDF (when in a `{table}`).

For the record, knitting to HTML and/or github .md is no problem: all characters come out as intended. So it does seem to be an issue with the papaja pipeline.

**Comment by mdingemanse** [comment]
Here's a minimal version:

`---
always_allow_html : yes
title             : "test"
shorttitle        : "test"

author: 
  - name          : "test"
    affiliation   : "1"
    corresponding : yes    # Define only one corresponding author
    address       : "A"
    email         : "B"

affiliation:
  - id            : "1"
    institution   : "A"

header-includes:
   - \usepackage[utf8]{inputenc}

author_note: |
  Add complete departmental affiliations for each author here. Each new line herein must be indented, like this line.

  Enter author note here.

abstract: |
  test.
  
keywords          : "test"
wordcount         : "NA"

figsintext        : yes
figurelist        : no
tablelist         : no
footnotelist      : no
lineno            : yes
mask              : no

class             : man
fig_caption       : true
output            : papaja::apa6_pdf
#output            : papaja::apa6_word
---

```{r load_packages, include = FALSE}
library("papaja")
```

If I knit an Rmd document that includes UTF-8 characters outside the ASCII range (for instance, phonetic symbols like ɛ or ɔ), they never survive the transition to PDF (with output: `papaja::apa6_pdf`) or to .docx (with output: `papaja::apa6_word`).

In the PDF, they are either invisible (when embedded in a paragraph) or show up as the unparsed codepoints (e.g. <U+0254>) in the PDF (when in a {table}).

\begin{table}[tbp]
\begin{center}
\begin{threeparttable}
\caption{Test: using IPA characters in table.}
\begin{tabular}{lllll}
\toprule
Stimuli.Number & \multicolumn{1}{c}{F1} & \multicolumn{1}{c}{F2} & \multicolumn{1}{c}{VowelCat} & \multicolumn{1}{c}{Graphcat}\\
\midrule
1 & 231 & 2324 & i & i\\
1 & 706 & 1833 & ɛ & e\\
3 & 222 & 774 &  ɔ & o\\
\bottomrule
\end{tabular}
\end{threeparttable}
\end{center}
\end{table}
`

**Comment by crsh** [comment]
If I try to knit your file I get the following error:

~~~
! Package inputenc Error: Unicode char ɛ (U+25B)
(inputenc)                not set up for use with LaTeX.

Error: Failed to compile knit-test.tex. See knit-test.log for more info.
Execution halted
~~~

I get the same error with `pdf_document()`, so this is not strictly an issue related to `papaja`. It's a LaTeX problem. If you are happy with the PDF output format try adding the following to the YAML front matter:

~~~yaml
header-includes:
  - \usepackage{tipa}
~~~

and use the [packages syntax](https://www.tug.org/tugboat/tb17-2/tb51rei.pdf)  to create the desired symbols:

~~~
If I knit an Rmd document that includes UTF-8 characters outside the ASCII range (for instance, phonetic symbols like \textipa{[E]} or \textipa{[O]}), they never survive the transition to PDF (with output: `papaja::apa6_pdf`) or to .docx (with output: `papaja::apa6_word`).
~~~

For Word documents it works on my machine, but I suspect you are working on a Windows computer and the issue of disappearing symbols may be related to #133, #159 and possibly #139. I have an idea how to fix this but it might be a while until I get around to it.

**Comment by crsh** [comment]
Hi @mdingemanse, I think I finally got a handle on this. As noted above, the phonetic symbols cannot be used directly in PDF documents. You can use the XeLaTeX or LuaLaTeX enginges to avoid the above encoding error, but the symbols won't show up in the document.

~~~yaml
output            :
  papaja::apa6_pdf:
    latex_engine: xelatex
~~~

So, I think you'll have to use the `tipa` package. The fix I pushed, however, ensures that the symbols show up in the word document. I hope this helps. Let me know if it works on your machine, when you find the time.

**Comment by witteerik** [comment]
Hi,
I've had the same problem. The reason the IPA characters do not show may be that they are lacking in the specified font. I solved the problem by temporarily change the font to Doulos SIL or Arial at the location of the phonetic character. To temporarily change the font, I did the following in the R-Markdown body:


\newfontfamily\PF{Doulos SIL} 


This is my phonetic characters [\begingroup \PF ʉː\endgroup] within square brackets.

Unfortunately, this solution does not work with tables, as the unicode fonts seem to be converted to a ASCII representation of its hexcode somewhere along the line. Any ideas on how to fix this in tables?
